commit b61f8ff9e0555e31fdb774244e68fa447c0bec4e (refs/changes/88/3888/1)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2018-04-30T18:57:44+00:00 (1 year, 5 months ago)
    
    MORAY-468 "close-handles" test does not account for stdout/stderr

diff --git a/test/close-handles.test.js b/test/close-handles.test.js
index 7cf2984..8e5f247 100644
--- a/test/close-handles.test.js
+++ b/test/close-handles.test.js
@@ -66,7 +66,9 @@ tape.test('client close actually closes sockets', function (t) {
         }
     ], function (err) {
         t.error(err);
-        t.equal(0, process._getActiveHandles().length);
+        t.deepEqual([], process._getActiveHandles().filter(function (h) {
+            return (h !== process.stderr && h !== process.stdout);
+        }));
         t.end();
     });
 });
